Audience Sync to TikTok (#4902)
braze-inc/braze-docs: _docs/_hidden/private_betas/pinterest_audience_sync.md

Your Pinterest connection will be applied at the Braze app group level. If your Pinterest admin removes you from your Pinterest Business Hub or access to the connected Pinterest accounts, Braze will detect an invalid token. As a result, your active Canvases using Pinterest Audience components will show errors, and Braze will not be able to sync users.

### Step 2: Configure your Canvas entry criteria

> To prevent sending users that have opted out of ads tracking and/or opted into the "Do Not Sell or Share" as per the [CCPA](https://oag.ca.gov/privacy/ccpa), marketers should ensure they are implementing the proper filters within their Canvas entry criteria.

Reviewer: Suggestion to remove `and/or` plus speak directly to the user 
```suggestion
To prevent sending to users that have opted out of ads tracking or opted into the "Do Not Sell or Share" as per the [CCPA](https://oag.ca.gov/privacy/ccpa) or both, ensure you are implementing the proper filters within your Canvas entry criteria.
```

If you have opted into collecting the [iOS IDFA through the Braze SDK](https://www.braze.com/docs/developer_guide/platform_integration_guides/ios/initial_sdk_setup/other_sdk_customizations/#optional-idfa-collection), you will be able to use the **Ads Tracking Enabled** filter. Set the value as `true` to only send users into Audience Sync destinations where they have opted in. 

![][16]{: style="max-width:75%;"}

If you collect opt-ins or opt-outs as Braze custom attributes, you should also include them within your Canvas entry criteria as a filter:

> ![][13]{: style="max-width:75%;"}

Author: ```suggestion
![A Canvas with an entry audience of "opted_in_marketing" equals "true".][13]{: style="max-width:75%;"}
```

> ![][13]{: style="max-width:75%;"}

Reviewer: @KellieHawks this seems to provide important info based on the context (filters), we should add appropriate alt text here or include this information as text in the surrounding area
